scripts/publish/prepublish.js

```diff
@@ -16,7 +16,7 @@ if (!isValidPackageName) {
 
 // Run babel with custom plugins to fix non-index internal imports and add .js extensions.
 const scriptPath = path.join(__dirname, 'replace-imports.sh');
-const args = [path.join(__dirname, '..', 'packages', packageName, 'lib')];
+const args = [path.join(__dirname, '..', 'packages', packageName, 'lib'), 'cjs'];
```

Comment:
Doesn't it still need a `'..'` like:


```suggestion
const args = [path.join(__dirname, '..', '..', 'packages', packageName, 'lib'), 'cjs'];
```

❓

Reply:
True, forgot the file was also moved